features/cart/shopping_cart/remove_product_from_cart_when_not_available.feature

```diff
@@ -0,0 +1,55 @@
+@shopping_cart
+Feature: Remove product from cart when not available
+    In order to avoid checkout of unavailable products
+    As a shop user
+    I want to be notified when a product in my cart is no longer available
+    And I want the unavailable product to be removed from my cart
+
+    Background:
+        Given the store operates on a single channel in "United States"
+
+    @ui
```

Comment:
```suggestion
    @api @ui
```

These scenarios apply to both UI and API. However, it would be good to add one `@no-api @ui` scenario that also checks the presence of the flash message.